pkg/provider/aws/knowledgebase/iam.go

```diff
@@ -141,6 +141,10 @@ var IamKB = knowledgebase.Build(
 	},
 	knowledgebase.EdgeBuilder[*resources.IamRole, *resources.OpenIdConnectProvider]{
 		Configure: func(role *resources.IamRole, oidc *resources.OpenIdConnectProvider, dag *core.ResourceGraph, data knowledgebase.EdgeData) error {
+			if strings.Contains(role.Name, "alb-controller") {
```

Comment:
Not sure how I feel about relying on the `Name` field programmatically. Probably fine for now, but maybe there's another way using dependencies or something that would be more robust.

Reply:
yeah so this is another case of not having the helm chart broken down, realistically we would want to be able to go directly to the service account name in here, which we cant do today from our kubernetes setup. Definitely will need to revisit this